From abf8f73655656d383de8c125b6d5b4b3a76f827b Mon Sep 17 00:00:00 2001
From: Saahil Tomar <quic_saahtoma@quicinc.com>
Date: Sun, 29 Oct 2023 00:41:53 +0530
Subject: [PATCH] arm64: boot: dts: qcom: Add MI01.7 (RDP473) dts Makefile
 support.

This RDP 473 (MI01.7) is based on RDP442 (MI01.3).
RDP473 consists of IPQ5332 + QCN6432 + QCN6432 + QCN9160(M+P+P+Y).

Change-Id: I25830ce0badb2a682b74220216118073aca50e33
Signed-off-by: Saahil Tomar <quic_saahtoma@quicinc.com>
---
 arch/arm/boot/dts/Makefile        | 1 +
 arch/arm64/boot/dts/qcom/Makefile | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm/boot/dts/Makefile b/arch/arm/boot/dts/Makefile
index fd5c83c23c5d..878f593fa84e 100644
--- a/arch/arm/boot/dts/Makefile
+++ b/arch/arm/boot/dts/Makefile
@@ -1080,6 +1080,7 @@ dtb-$(CONFIG_ARCH_QCOM) += \
 	ipq5332-rdp446.dtb \
 	ipq5332-rdp468.dtb \
 	ipq5332-rdp472.dtb \
+	ipq5332-rdp473.dtb \
 	ipq5332-rdp474.dtb \
 	ipq5332-rdp477.dtb \
 	ipq5332-rdp478.dtb \
diff --git a/arch/arm64/boot/dts/qcom/Makefile b/arch/arm64/boot/dts/qcom/Makefile
index e7b90f8a4195..db0ba21106d3 100644
--- a/arch/arm64/boot/dts/qcom/Makefile
+++ b/arch/arm64/boot/dts/qcom/Makefile
@@ -10,6 +10,7 @@ dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp442.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp446.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp468.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp472.dtb
+dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp473.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp474.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp477.dtb
 dtb-$(CONFIG_ARCH_QCOM)	+= ipq5332-rdp478.dtb
-- 
2.34.1

